### PR TITLE
Add Korean translation and small change on all languages

### PR DESCRIPTION
**Pull request by Johnmacrocraft**
This pull request adds _*Korean translation*_ and makes _*a small change on all languages*_.

P.S: When I tested my Korean translation, server gave me a message "Warning: syntax error, unexpected '!' in (My directory)\messages.ini on line 17
 in (My directory)\Skin.php on line 151". I tried to fix that, but I couldn't fix it. So you need to fix it yourself (Sorry).

### CODE REVIEW

**Comment by Johnmacrocraft** [comment]
@SalmonDE, @KairusDarkSeeker, @Aericio Will you update version with this pull request, too?

**Comment by Aericio** [comment]
No, because changing version doesn't update config or language files.

File: resources/kor.ini [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+SenderMustBePlayer = '이 명령어는 콘솔로 실행할 수 없습니다!'
+[ChangeSkin]
+PlayerNotFound = '{player}(을)를 찾지 못했습니다!'
+SkinChanged = '스킨이 변경되었습니다!'
+SkinNotFound = '죄송합니다! 이 스킨은 존재하지 않습니다! 다음을 입력했는지 확인하세요: §b{skin} '
+SkinInCheck = '잠시만 기다려 주세요, 현재 당신의 스킨을 확인하고 있습니다.'
+SkinNameMissing = '스킨 이름이 없습니다!'
+[ChangeCape]
+CapeChanged = '망토가 변경되었습니다!'
+CapeNotFound = 망토를 찾지 못했습니다!'
```
Comment (Aericio) [inline resources/kor.ini]: You missed a ' here.

Comment (Johnmacrocraft) [inline resources/kor.ini]: WT*, why I didn't take a look as well, thank you for letting me figure out the problem.

**Comment by Johnmacrocraft** [comment]
@Aericio Maybe I'll need to create new pull request because I accidentally named first commit as something like update, not add

**Comment by Aericio** [comment]
That's fine, I'll squash it.

**Comment by Aericio** [comment]
That's fine, I'll squash it instead.

**Pull request merged**